File: StudentCredit.Application/Credits/Dtos/CreditApplicationDto.cs
using StudentCredit.Application.Nomenclatures.Dtos;
using StudentCredit.Data.Common.Enums;
using StudentCredit.Infrastructure.Helpers.Dtos;

namespace StudentCredit.Application.Credits.Dtos
{
	public class CreditApplicationDto
	{
		public int Id { get; set; }

		public NomenclatureDto ApplicationType { get; set; }

		public DateTime BlankDate { get; set; }

		public BankNomenclatureDto Bank { get; set; }

		public CommitState CommitState { get; set; }
	}
}

File: WebApplication/src/modules/application/credit/models/credit-application.dto.ts
import { NomenclatureDto } from "src/infrastructure/models/nomenclature.dto";
import { CommitState } from "../../common/enums/commit-state.enum";
import { BankNomenclatureDto } from "src/modules/nomenclature/common/models/bank-nomenclature.dto";

export class CreditApplicationDto {
  id: number;
  applicationType: NomenclatureDto;
  blankDate: Date;
  bank: BankNomenclatureDto;
  commitState: CommitState;
}